aether/sdk/conf/settings.py

```diff
@@ -256,6 +256,19 @@ def get_required(name):
 # A slick ORM cache with automatic granular event-driven invalidation.
 
 if DJANGO_USE_CACHE:
+
+    # Cache Redis Sessions using django-redis-sessions
+    SESSION_REDIS = {
+        'host': REDIS_HOST,
+        'port': REDIS_PORT,
+        'db': REDIS_DB + 1,
```

Comment:
same redis db as cacheops, isn't there any conflict?

Reply:
In that case us the variable `REDIS_DB_CACHEOPS` to ensure that both use the same database.

Reply:
:+1: